framegear/contexts/TextInputs.tsx

```diff
@@ -0,0 +1,34 @@
+import React, { createContext, useState, useContext, ReactNode } from 'react';
+
+type TextInputsContextType = {
```

Comment:
At some point we want to start organize the types in a `types.ts` file.

Reply:
Yeah I was wondering about that. And also moving the `RedirectModalContext` into contexts. 

I'll do a little clean-up PR in the next couple days with some of those chores.